manifests/0000_50_olm_00-operatorconditions.crd.yaml

```diff
@@ -0,0 +1,199 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: operatorconditions.operators.coreos.com
+spec:
+  group: operators.coreos.com
+  names:
+    kind: OperatorCondition
+    listKind: OperatorConditionList
+    plural: operatorconditions
+    singular: operatorcondition
+  scope: Namespaced
```

Comment:
I thought this was supposed to be a cluster scoped resource?

Reply:
@kevinrizza and I discussed this earlier and we decided to keep it namespaced.

Reply:
Can you record the reasoning either here or in the enhancement so that we don't forget?

Reply:
I'll update the enhancement.